addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml

```diff
@@ -2,14 +2,23 @@
 <odoo>
 
     <data noupdate="1">
+        <record id="account_1003" model="account.account.template">
+            <field name="name">转让帐户</field>
+            <field name="code">1003</field>
+            <field name="reconcile" eval="True" />
+            <field name="user_type_id" ref="account.data_account_type_current_assets" />
+        </record>
         <!-- Chart template -->
         <record id="l10n_chart_china_standard_business" model="account.chart.template">
             <field name="name">中国会计科目表 （财会[2006]3号《企业会计准则》</field>
```

Comment:
[TYPO] There should be a '）' at the end.  :-)

Reply:
You are very careful～

Reply:
Good job.@joshuajan